lib/checks/shared/doc-has-single-title-evaluate.js

```diff
@@ -0,0 +1,6 @@
+function docHasSingleTitleEvaluate() {
+  var titles = document.getElementsByTagName('title');
+  return titles.length === 1;
```

Comment:
There's already a rule that checks there is a single title. We don't need to fail missing titles twice.

```suggestion
  return titles.length >= 1;
```

Reply:
Which rule is that? `doc-has-title`? It checks the document API, not the html/DOM. So if I added `>= 1` here we'd be allowing more than 1 title to exist in the html, which is bad practice

Reply:
I could edit/update `doc-has-title` to assert both the document api and the markup, but this might cause some breaking changes as it edits a rule

Reply:
I think adding multiple checks to `doc-has-title` would be a bit easier vs separate rule

Reply:
@cmswalker Wouldn't work I'm afraid. The `document-title` rule is required for WCAG, whereas this would be a best practice. We have to report them differently, we can only do that by putting them into separate rules.